.gitignore

```diff
@@ -8,4 +8,6 @@
 *.rsc
 *.lk
 *.int
-*.backup
\ No newline at end of file
+*.backup
+config/admins.txt
```

Comment:
no, I said to add it to you local repo, that has no relation to you adding it to the mainline